Multisig/Logic/Models/Network.swift

```diff
@@ -91,7 +91,7 @@ extension Network {
     static func create(_ networkInfo: SCGModels.Network) throws -> Network {
         try Network.create(chainId: networkInfo.chainId,
                      chainName: networkInfo.chainName,
-                     rpcUrl: networkInfo.authenticatedRpcUrl,
+                     rpcUrl: networkInfo.rpcUrl,
```

Comment:
why did you change it here?

Reply:
Because by the old why we store Infura key in the database as part of rpc url